src/components/Search/index.tsx

```diff
@@ -197,10 +204,18 @@ const Results = connectStateResults(
   }
 )
 
-const Search = ({ handleSearchSelect, useKeyboardShortcuts }) => {
+interface ISearchProps {
+  handleSearchSelect: () => void
+  useKeyboardShortcuts: boolean
+}
+
+const Search: React.FC<ISearchProps> = ({
+  handleSearchSelect,
+  useKeyboardShortcuts,
+}) => {
   const intl = useIntl()
-  const containerRef = useRef()
-  const inputRef = useRef()
+  const containerRef = useRef<HTMLDivElement>(null)
+  const inputRef = useRef<HTMLInputElement>(null)
   const [query, setQuery] = useState(``)
   const [focus, setFocus] = useState(false)
   const algoliaClient = algoliasearch(
```

Comment:
Lets install the types for algolia. `yarn add -D @types/react-instantsearch-dom`